userspace/libscap/scap-int.h

```diff
@@ -415,13 +377,13 @@ extern bool validate_info_table_size();
 //
 // udig stuff
 //
-int32_t udig_begin_capture(scap_t* handle, char *error);
-void udig_start_capture(scap_t* handle);
-void udig_stop_capture(scap_t* handle);
-void udig_end_capture(scap_t* handle);
-uint32_t udig_set_snaplen(scap_t* handle, uint32_t snaplen);
-int32_t udig_stop_dropping_mode(scap_t* handle);
-int32_t udig_start_dropping_mode(scap_t* handle, uint32_t sampling_ratio);
+int32_t udig_begin_capture(struct scap_engine_handle engine, char *error);
```

Comment:
Can't we define these in udig.h?

Reply:
I haven't focused on cleaning up scap-int.h yet. Once we're done, I'd hope most of it goes away completely.

Reply:
Makes sense, thanks!